iroha/docs/src/main.rs

```diff
@@ -7,12 +7,12 @@ use std::{
     io::{stdout, BufWriter, Result, Write},
 };
 
+use eyre::Context;
 use iroha::config::Configuration;
 use iroha_config::Configurable;
-use iroha_error::WrapErr;
 use serde_json::{Map, Value};
 
-fn main() -> iroha_error::Result<()> {
+fn main() -> eyre::Result<()> {
```

Comment:
`color_eyre` might be a good pick here. For error reporting

Reply:
color_eyre reexports eyre::Result.
iroha_docs doesn't depend on color-eyre.
Do you want to me to add a dependency?

Reply:
Yep, i think you should add one. I guess we simply forgot to add report hook here